addons/account/models/account_move_line.py

```diff
@@ -1561,6 +1558,20 @@ def write(self, vals):
 
         return result
 
+    def _parse_flush_fnames(self, fnames):
+        if fnames and {'balance', 'amount_currency'} & set(fnames):
+            # flush the amount currency to avoid triggering check_amount_currency_balance_sign
+            fnames = {'balance', 'amount_currency'} | set(fnames)
+        return fnames
+
+    def flush_recordset(self, fnames=None):
+        # EXTENDS models
```

Comment:
I know some people actually like these, but I find it really useless.
An IDE is way more powerful than this, and the fact that `super()` is called is enough to know that it is an extension...